README.md

```diff
@@ -1805,9 +1805,11 @@ The following extractors use this feature:
 * `max_comments`: Maximum number of comments to extract - default is `120`
 
 #### tiktok
-* `api_hostname`: Hostname to use for mobile API requests, e.g. `api-h2.tiktokv.com`
-* `app_version`: App version to call mobile APIs with - should be set along with `manifest_app_version`, e.g. `20.2.1`
-* `manifest_app_version`: Numeric app version to call mobile APIs with, e.g. `221`
+* `iid`: Install ID of the mobile app required for API requests, e.g. `1234567890123456789`
+* `app_name`: Mobile app name to use for API requests, e.g. `trill`
```

Comment:
Since these are always meant to be passed together,  how about `app=name.version.manifest` like `tiktok:app=thrill.1.2.3.4.56789`

Reply:
I think it would make more sense to pair `app_name` and `iid` together instead of app name and versions.

`app_version` and `manifest_app_version` do not seem to be checked against the `iid`. The current app/manifest version works with both `musical_ly` and `trill` and any `iid` that I try passing. I am keeping those two extractor-args as a contingency plan for future breakage

Reply:
In that case, you can keep them separate